taxi/views.py

```diff
@@ -87,3 +88,47 @@
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(View):
+    def get(self, request):
+        form = DriverCreationForm()
+        return render(request,
+                      "taxi/driver_list.html", {"form": form})
+
+    def post(self, request):
+        form = DriverCreationForm(request.POST)
+        if form.is_valid():
+            form.save()
+            return redirect("taxi:driver-list")
+        return render(request,
+                      "taxi/driver_list.html", {"form": form})
```

Comment:
Similar to the GET method, the POST method in `DriverCreateView` should render a more appropriate template, such as 'taxi/driver_form.html', instead of 'taxi/driver_list.html'.